src/main/java/seedu/address/commons/core/Alias.java

```diff
@@ -30,6 +30,9 @@ public Alias() {
 
     /**
      * Creates an Alias object.
+     * All fields should not be null.
+     *
+     * @throws NullPointerException if the input is null.
```

Comment:
Perhaps the first letter of the `@` tag description should be capitalized. That is,

` @throws NullPointerException If the input is null.`

Saw this in most other files.